index.html

```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <h1 style="color:blue"> Feel the Power of Typography</h1>
+        <link rel="stylesheet" href="/main.css">
+
     </head>
     <body>
         <div id="header"> 
+            <h1> Feel the Power of Typography</h1>
             <div style="margin-bottom: -300px;">
```

Comment:
jak już masz podpięty plik ze stylami css to nie korzystałbym z inline styles tylko przeniósłbym ten margin-bottom do pliku css